app/src/store/features/userActions.js

```diff
@@ -113,3 +113,7 @@ export const updateWorkspaceOnboardingTeamDetails = (prevState, action) => {
 export const updateIsCommandBarOpen = (prevState, action) => {
   prevState.misc.nonPersist.isCommandBarOpen = action.payload;
 };
+
+export const updateLastSeenInvites = (prevState, action) => {
+  prevState.misc.persist.lastSeenInvites = action.payload;
+};
```

Comment:
Persistence of this isn't mentioned in the PRD. Do we need to do this? Makes things complicated for now